### ISSUE TITLE
Clean up what notifications are shown when user opens Blazor project in VS Code

### ISSUE THREAD

**Issue by rynowak** [dotnet/aspnetcore#23832]
### Describe the bug

VS Code is showing me this notification many times after opening a repo.

<img width="542" alt="image" src="https://user-images.githubusercontent.com/1430011/87102089-afe70780-c205-11ea-93ad-56877888c405.png">

### To Reproduce

Unfortunately I cannot provide access to the repo. I opened VS code in the root of a git repository containing a bunch of .NET projects, and one of them is Blazor WASM.

I'd really prefer to see this when I try to launch/debug a Blazor WASM project or not at all. VS Code already has too much notification spam.

### Further technical details
- On macOS
- C# extension 1.22.1

**Comment by NTaylorMullen** [comment]
@rynowak I assume you don't want to setup the additional steps for Blazor Wasm debugging right? Also, this only pops up once upon opening VSCode right? or are you saying that it pops up repeatedly/many times in the same session?

@captainsafia what do you think about adding a "snooze"/ don't bother me anymore option that's cached?

**Comment by captainsafia** [comment]
I believe this kind thing should only happen when you open VS Code. This message pops up when the O# extension does workspace discovery to determine what kinds of .NET projects you have ([ref](https://github.com/OmniSharp/omnisharp-vscode/blob/6a73ed29f162b8fb587da3d4ae925a94da6d459d/src/omnisharp/utils.ts#L91)).

We can implement some "show this only once for this VS Code instance" logic on the O# extension to resolve this issue.

**Comment by rynowak** [comment]
>  assume you don't want to setup the additional steps for Blazor Wasm debugging right? 

I did not do this, because I wasn't working with the Blazor project. I believe this repo has a Blazor WASM project that was created using one of the pre-releases, and some corresponding tasks.json stuff.


>Also, this only pops up once upon opening VSCode right? or are you saying that it pops up repeatedly/many times in the same session?

> **VS Code is showing me this notification many times after opening a repo.**

The notification kept coming back after I dismissed it, for a couple of minutes.

**Comment by captainsafia** [comment]
I just ran into this will working on the aspnetcore repo. 

I had `src/Components` open for about an hour when the prompt popped up. Based on the logs, the O# extension was analyzing the workspace at the same time so that might be a breadcrumb for this issue.

**Comment by captainsafia** [comment]
Per the mentioned issue above, now that we no longer require users to have the nightly JS debug extension, we can actually get rid of this additional setup warning all togehter.

**Comment by captainsafia** [comment]
Commandeering this issue to address all notification related problems in VS Code:

- Need to remove the "additional setup" notification
- Need to make sure the required assets prompt is displayed

@mkArtakMSFT Can we track this for RC1? Should be low-cost.

**Comment by captainsafia** [comment]
A fix for this has been merged into the C# extension.
